src/include/pgagroal.h

```diff
@@ -228,6 +228,30 @@ struct prometheus_connection
    atomic_ullong query_count;           /**< The number of queries per connection */
 } __attribute__ ((aligned (64)));
 
+/**
+ * Max size of the cache (in bytes).
+ * If the cache request exceeds this size
+ * the caching should be aborted in some way.
+ */
+#define PROMETHEUS_MAX_CACHE_SIZE 8192
```

Comment:
This should be something like 1M - e.g. 1024*1024

We will need a PROMETHEUS_DEFAULT_CACHE_SIZE as well at 256 * 1024

Reply:
Agree. I was not sure about the cache size to use.